packages/api-client-core/package.json

```diff
@@ -28,6 +28,7 @@
     "prerelease": "gitpkg publish"
   },
   "dependencies": {
+    "@manufac/repeater": "^4.0.6",
```

Comment:
Can you share the source for this package with us? I'm fine either way if you want to keep it under your github account or move it to ours, just want to make sure we can patch it further once the engagement is done

Reply:
Added you to the repo. Planning to add all that code into this same monorepo in a subsequent PRs. They are using a different version of TS which was causing some compilation errors with the TS config used here.